CMakeLists.txt

```diff
@@ -284,6 +294,18 @@ if (OPM_ENABLE_PYTHON)
     set( _rpath_arg "")
   endif()
 
+  if (OPM_ENABLE_PYTHON_DYNAMIC_LINKING)
+    set(opm-common_PYTHON_LINKAGE "'-Bdynamic'")
+  else()
+    set(opm-common_PYTHON_LINKAGE "'-Bstatic', '-static-libstdc++'")
+  endif()
+
+  set(opm-common_PYTHON_PACKAGE_VERSION ${OPM_PYTHON_PACKAGE_VERSION_TAG})
+
+  # Generate versioned setup.py
+  configure_file (${PROJECT_SOURCE_DIR}/python/setup.py.in ${PROJECT_BINARY_DIR}/python/setup.py)
+  configure_file (${PROJECT_SOURCE_DIR}/python/README.md ${PROJECT_BINARY_DIR}/python/README.md COPYONLY)
```

Comment:
If it is purely a copy I guess the `file(COPY ...)` could be used?

Reply:
sure!